src/Umbraco.Web.Common/Security/ConfigureIISServerOptions.cs

```diff
@@ -1,18 +0,0 @@
-using Microsoft.AspNetCore.Builder;
-using Microsoft.Extensions.Options;
-using Umbraco.Cms.Core.Configuration.Models;
-
-namespace Umbraco.Cms.Web.Common.Security
-{
-    public class ConfigureIISServerOptions : IConfigureOptions<IISServerOptions>
```

Comment:
Isn't it a breaking change to remove this class entirely, even if it's now obsolete, shouldn't we instead obsolete it, mentioning that this doesn't work? Maybe linking to, or mentioning the docs explaining how you can do it instead? At least no one will have compile errors then, but are aware of the runtime issues?

Granted this will only happen if someone is manually calling `services.ConfigureOptions<ConfigureIISServerOptions>();` but still?

Reply:
Great catch, didn't even think about it cause it did not work, will reinstate this and obsolete instead 👍